apps/web/src/containers/TradingContest/components/contest-sign-up-steps/community-partners.tsx

```diff
@@ -0,0 +1,185 @@
+import { styled, Box, useTheme } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+import { useSideDrawerManager } from '@notional-finance/side-drawer';
+import { SETTINGS_SIDE_DRAWERS } from '@notional-finance/util';
+import {
+  TitleText,
+  ContestBodyText,
+  StepContainer,
+} from '../contest-shared-elements/contest-shared-elements';
+import { ContestButtonBar } from '../contest-button-bar/contest-button-bar';
+import Llama from './assets/Llama.svg';
+import Cryptotesters from './assets/Cryptotesters.svg';
+import L2DAO from './assets/L2DAO.svg';
+import { useNotionalContext } from '@notional-finance/notionable-hooks';
+
+const imgData = {
```

Comment:
Is it possible to put these icons with the community membership code? No worries if not since the SVGs might not compile properly

Reply:
I didn't think adding in the assets to notionable really made sense. I am fine with leaving the images here.